rfcs/moderation-log.md

```diff
@@ -0,0 +1,98 @@
+concerns, though, such as privacy, and should be designed such that it does not
+encourage further harassment or defamation of anyone involved.
+
+### Keeping a Balance
+
+This RFC intends to balance the following specific concerns:
+
+* Openness about process: people should have a good idea of exactly how often and in what sorts of situations the Moderation Team has gotten involved.
+* Recordkeeping: For situations such as repeated violations (or lack thereof), or future analysis that can help the moderation process improve or change based on actual data.
+* Privacy/Sensitivity: Some details about moderation are often private, and should stay that way.
+* Prevent Further Harm: The log must not be a source of threat for users. It should not be seen or treated as a wall of shame of any sort.
+
+### Log Structure
+
+The log should be a spreadsheet with public read access, and edit access only
+available to active members of the Moderation Team. Any time an event happens,
+the log should be updated as soon as possible by the Moderation Team members
+involved.
+
+The log will include the following columns:
+
+* Date - date of the *enforcement action* (not the event itself)
+* Action Taken - official action taken by the moderation team, in accordance with the moderation process.
+* Venue - Where the event that triggered moderation happened: Github, Discord, Conference, etc.
+* Moderator(s) - Moderation Team member or members who took the action.
+* URL - If the event happened on a URL-linkable **PUBLIC** location (such as Github), the URL of the issue or issue comment in question.
+* Details - Freeform description of the event, with specific parties omitted.
+  This should be as dry and neutral as possible. This should not include names
+  of any involved parties except Moderators unless the event is publicly
+  documented elsewhere (such as on GitHub).
+
+### Scrubbing the Log
+
+In some cases, log entries may include personal details or other information
+which may unfairly target one or more of the people involved. In these cases,
+general community members may contact the Moderation Team through either public
+or private channels to request a log line be scrubbed of details.
+
+When a log line is scrubbed, the URL and Details field should be cleared and the
+Details field replaced with `[scrubbed on request]`. Unless circumstances are
+considered to be extreme enough, the rest of the line, including the date, the
+action, the venue, and the involved moderators, should be left intact. The
+requestor may also ask that only certain details be scrubbed from the
+description.
+
+Log lines should only be scrubbed when there is reason to believe that its
+existence would:
+
+1. Cause harm to a community member, either physical, emotional, or career-wise.
+2. Could be considered defamatory.
+3. Reveals private details or information.
+4. Incite targeted harassment in any way to any of the involved parties.
```

Comment:
this might be self-evident, but should “when legally required” be listed here?

Reply:
Yeah, that's a good idea.